cli/dcoscli/node/main.py

```diff
@@ -23,7 +23,6 @@
     --version               Show version
 """
 
-import os
```

Comment:
Writing the comment here but it applies to the `--config-file` option above. For my own edification why do we have a special flag and not make it part of `--option`.

Also, Is it too late to change it to `--ssh-option` instead of `--option`?